doc/rados/operations/crush-map-edits.rst

```diff
@@ -32,193 +33,199 @@ Pool Values`_.
-edit it.
+Ceph outputs (``-o``) a compiled CRUSH map to the filename that you have
+specified. Because the CRUSH map is in a compiled form, you must first
+decompile it before you can edit it.
 
 .. _decompilecrushmap:
 
-Decompile a CRUSH Map
----------------------
+Decompile the CRUSH Map
+-----------------------
 
-To decompile a CRUSH map, execute the following:
+To decompile the CRUSH map, run a command of the following form:
 
 .. prompt:: bash $
 
-	crushtool -d {compiled-crushmap-filename} -o {decompiled-crushmap-filename}
+    crushtool -d {compiled-crushmap-filename} -o {decompiled-crushmap-filename}
 
 .. _compilecrushmap:
 
-Recompile a CRUSH Map
----------------------
+Recompile the CRUSH Map
+-----------------------
 
-To compile a CRUSH map, execute the following:
+To compile the CRUSH map, run a command of the following form:
 
 .. prompt:: bash $
 
-	crushtool -c {decompiled-crushmap-filename} -o {compiled-crushmap-filename}
+    crushtool -c {decompiled-crushmap-filename} -o {compiled-crushmap-filename}
 
 .. _setcrushmap:
 
 Set the CRUSH Map
 -----------------
 
-To set the CRUSH map for your cluster, execute the following:
+To set the CRUSH map for your cluster, run a command of the following form:
 
 .. prompt:: bash $
 
-	ceph osd setcrushmap -i {compiled-crushmap-filename}
+    ceph osd setcrushmap -i {compiled-crushmap-filename}
 
-Ceph will load (-i) a compiled CRUSH map from the filename you specified.
+Ceph loads (``-i``) a compiled CRUSH map from the filename that you have
+specified.
 
 Sections
 --------
 
-There are six main sections to a CRUSH Map.
+A CRUSH map has six main sections:
 
 #. **tunables:** The preamble at the top of the map describes any *tunables*
-   that differ from the historical / legacy CRUSH behavior. These
-   correct for old bugs, optimizations, or other changes that have
-   been made over the years to improve CRUSH's behavior.
+   that are not a part of legacy CRUSH behavior. These tunables correct for old
+   bugs, optimizations, or other changes that have been made over the years to
+   improve CRUSH's behavior.
 
 #. **devices:** Devices are individual OSDs that store data.
 
-#. **types**: Bucket ``types`` define the types of buckets used in
-   your CRUSH hierarchy. Buckets consist of a hierarchical aggregation
-   of storage locations (e.g., rows, racks, chassis, hosts, etc.) and
-   their assigned weights.
+#. **types**: Bucket ``types`` define the types of buckets that are used in
+   your CRUSH hierarchy. 
 
-#. **buckets:** Once you define bucket types, you must define each node
-   in the hierarchy, its type, and which devices or other nodes it
-   contains.
+#. **buckets:** Buckets consist of a hierarchical aggregation of storage
+   locations (for example, rows, racks, chassis, hosts) and their assigned
+   weights. After you have defined bucket ``types``, you must define each node
+   in the hierarchy, its type, and which devices or other nodes it contains.
```

Comment:
Maybe phrase in terms that the CRUSH map must define these things, so the gentle reader doesn't think they need to build it from scratch.

Reply:
Accepted.